routes/celebrities.routes.js

```diff
@@ -0,0 +1,34 @@
+const router = require("express").Router();
+const Celebrities = require('/models/celebrities.model')
```

Comment:
Here, the error on the terminal was here,
You should write like this: const Celebrity = require("../models/celebrities.model")

Reply:
so it takes ..? sometimes I find I do it and it gives me issues and sometimes I don't and its fine. I will add them back in and see if it works, thanks